XCUITests/NavigationTest.swift

```diff
@@ -171,12 +171,12 @@ class NavigationTest: BaseTestCase {
         goToDesktopFromMobile.tap()
```

Comment:
Should this use 

```swift
navigator.toggleOff(userState.requestDesktopState, forAction: Action.ToggleRequestDesktopSite)
```

Reply:
I am afraid in this case it is not possible because we are changing to desktop from the website, using an element there, not from the menu options given